operations/observability/mixins/meta/rules/login-slo.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
     prometheus: k8s
     role: alert-rules
   name: login-slo-monitoring-rules
+  namespace: monitoring-satellite
```

Comment:
```suggestion
```

Hey, we intentionally removed that pretty recently. We want to deploy those alerting rules in different namespaces during the transition period of alerting locally -> alerting in monitoring-central

Indeed you are correct that our intention is to let ArgoCD handle the default namespace. If we deploy those rules with satellite, ArgoCD will install it in `monitoring-satellite` namespace. If we deploy them to monitoring-central, ArgoCD will do the same for the `monitoring-central` namespace.

The reason we can't keep a namespace in the rules definition is that ArgoCD decides only the default namespace, which means that it won't change the namespace of resources that have that set already.

Reply:
I removed the commit that added the namespaces to the rules, and used kubens to temporarily set the default namespace (see https://github.com/gitpod-io/gitpod/pull/14505/commits/f550733dd228a473e7d7569da1fa93bf8f2a0b85)

With this change it the Werft job installs monitoring-satellite correctly and it appears the resources are added to the correct namespace

```
kubectl get prometheusrules.monitoring.coreos.com --all-namespaces
NAMESPACE              NAME                                     AGE
monitoring-satellite   cardinality-analysis-monitoring-rules    14m
monitoring-satellite   grafana-rules                            14m
monitoring-satellite   kube-prometheus-rules                    14m
monitoring-satellite   login-slo-monitoring-rules               14m
monitoring-satellite   messagebus-monitoring-rules              14m
monitoring-satellite   meta-nodes-monitoring-rules              14m
monitoring-satellite   openvsx-proxy-monitoring-rules           14m
monitoring-satellite   server-monitoring-rules                  14m
monitoring-satellite   ssh-gateway-monitoring-rules             14m
monitoring-satellite   usage-monitoring-rules                   14m
monitoring-satellite   workspace-failure-slo-monitoring-rules   14m
monitoring-satellite   workspace-monitoring-rules               14m
monitoring-satellite   workspace-nodes-monitoring-rules         14m
monitoring-satellite   ws-daemon-monitoring-rules               14m
monitoring-satellite   ws-manager-monitoring-rules              14m
```